src/nanorc/node.py

```diff
@@ -507,9 +507,26 @@ def _on_enter_callback(self, event):
                 done = []
                 for child_node in appset:
                     if not child_node.included: continue
-
-                    if not child_node.sup.desc.proc.is_alive() or not child_node.sup.commander.ping():
+                    IsAlive = child_node.sup.desc.proc.is_alive()
```

Comment:
use snake case variables like in the rest of the code, please 😉

Reply:
changed to is_alive